ProjectSidewalk/SidewalkWebpage
# completed config table for each city's database.

Resolves #2772 

Implemented a table that holds all of the relevant columns for a respective city's configuration such as latitude, offset hour, tutorial street edge id to easily update them. Had to include some funny SQL in 194.sql to have config be a one row table.

NOTE: Before approving, there is one comment with "- Dylan" to confirm if this should use ConfigTable or not.

##### Testing instructions
1. open choice for viewing table data (pgadmin >)
2. go into the config table and confirm if the data matches up. can also confirm this by looking at 194.sql.

##### Things to check before submitting the PR
